src/Service/Startup.cs

```diff
@@ -471,6 +492,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, RuntimeC
             });
         }
 
+        /// <summary>
+        /// Evicts the GraphQL schema from the request executor resolver.
+        /// </summary>
+        private static void EvictGraphQLSchema(IRequestExecutorResolver requestExecutorResolver)
+        {
+            Console.WriteLine("Evicting old GraphQL schema.");
+            requestExecutorResolver.EvictRequestExecutor();
```

Comment:
Great find! To evict, then refresh the schema. This was a difficult issue to resolve!!